src/ontology/uberon-edit.obo

```diff
@@ -221587,6 +221587,1077 @@ intersection_of: in_left_side_of UBERON:0010709 ! pelvic complex
+
+[Term]
+id: UBERON:8310002
+name: urethral seam
+def: "A solid epithelial cord present deep to the skin along the ventral midline of the penis. Represents a remnant of a developmental process that closes the urethra." [doi:10.1007/s004410000345, RRID:SCR_018709]
+synonym: "urethral raphe" EXACT []
+synonym: "ventral seam" BROAD []
+xref: EMAPA:36421
+xref: ILX:0738407
+is_a: UBERON:0000061 ! anatomical structure
+
+[Term]
+id: UBERON:8310003
+name: anterior subdiaphragmatic vagus nerve
+def: "A nerve formed by the convergence of nerve bundles of the esophageal plexus that arise primarily from the left vagus nerve." [doi:10.1016/B978-0-12-800898-0.00010-5, RRID:SCR_018709]
+xref: ILX:0738436
+is_a: UBERON:0001759 ! vagus nerve
+
+[Term]
+id: UBERON:8310004
+name: interscapular brown adipose tissue
+def: "A collection of brown adipose tissue between the medial borders of the scapulae." [RRID:SCR_018709, wikipedia:Brown_adipose_tissue]
+synonym: "iBAT" EXACT []
+xref: ILX:0738444
+xref: MA:0003062
+is_a: UBERON:0001348 ! brown adipose tissue
+
+[Term]
+id: UBERON:8310005
+name: ansa subclavia
+def: "A nerve cord that connects the middle cervical and inferior cervical sympathetic ganglia, forming a loop around the subclavian artery." [PMID:18828096]
+synonym: "Subclavian loop" EXACT []
+xref: FMA:6582
+xref: ILX:0739241
+is_a: UBERON:0001990 ! middle cervical ganglion
+
+[Term]
+id: UBERON:8310006
+name: inner submucosal nerve plexus
+def: "A sparse network of ganglia on the luminal side of the submucosa of the intestine, close to the muscularis mucosae muscle, in larger mammals." [RRID:SCR_018709]
+synonym: "inner submucosal plexus" EXACT []
+xref: ILX:0777077
+is_a: UBERON:0005304 ! submucous nerve plexus
+
+[Term]
+id: UBERON:8310007
+name: outer submucosal nerve plexus
+def: "A dense network of ganglia in the submucosa, close to the circular muscle of the intestine in larger mammals." [RRID:SCR_018709]
+synonym: "outer submucosal plexus" EXACT []
+xref: ILX:0777078
+is_a: UBERON:0005304 ! submucous nerve plexus
+
+[Term]
+id: UBERON:8310008
+name: seventh lumbar spinal cord segment
+def: "The segment of the spinal cord that gives rise to the dorsal and ventral roots that coalesce to form the seventh lumbar spinal nerve which exits the spinal column below the seventh lumbar vertebra in most mammals. Present in some animals such as pig, monkey, cat. Not present in humans." [doi:10.1002/ar.24054, doi:10.1016/B978-0-12-369497-3.10013-5, RRID:SCR_018709]
+synonym: "L7 segment" EXACT []
+synonym: "L7 spinal cord segment" EXACT []
+xref: ILX:0793850
+is_a: UBERON:0007716 ! lumbar subsegment of spinal cord
+
+[Term]
+id: UBERON:8310009
+name: coccygeal spinal cord
+def: "The most caudal segment of the spinal cord, which gives rise to the coccygeal dorsal and ventral roots." [PMID:31751060, RRID:SCR_018709]
+synonym: "coccygeal spinal cord segment" EXACT []
+xref: ILX:0740051
+xref: ilx:swanson/uris/neuroanatomical-terminology/terms/56
```

Comment:
I am not sure how to change this, would it be possible to have the URL? That might help. Thanks!

Reply:
I can expand all these curies to iris, if that would help.

Reply:
Thank you, only this one please, all the others are fine.